### PR TITLE
hyperv: Fixes for info, inspect tests 

### PR DESCRIPTION
**Pull request by ashley-cui**
Add podman socket info to machine inspect
Fix podman machine info test for hyperV 
Update machine tests README 

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by baude** [comment]
lgtm

File: pkg/machine/e2e/README.md [hunk 1]
```diff
@@ -14,13 +14,15 @@ Note: you must not have any machines defined before running tests
 1. Open a powershell as admin
 1. $env:CONTAINERS_MACHINE_PROVIDER="hyperv"
 1. $env:MACHINE_IMAGE="https://fedorapeople.org/groups/podman/testing/hyperv/fedora-coreos-38.20230830.dev.0-hyperv.x86_64.vhdx.zip"
```
Comment (mheon) [inline pkg/machine/e2e/README.md:16]: @baude This still necessary?

**Comment by mheon** [comment]
LGTM, one comment I'd like answered before merge

File: pkg/machine/e2e/README.md (hunk 1 above)
Comment (baude) [reply pkg/machine/e2e/README.md:16]: In reality, no; we can fix that down the road.

**Comment by TomSweeneyRedHat** [comment]
LGTM
boltdb test threw up with a timeout, restarting.

**Pull request merged**